client/src/components/RequestAccess/RequestAccess.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
```

Comment:
Will we have a `/request-access` page for all portals @jmartinez-tacc @rstijerina even though its only applicable for 3DEM or similar portals who similarly lock access at the WSO2/tapis-tenant?  I guess my question is will we just always have the page and only have a CMS-based link to it in certain portals? or do we also want to disable this route via a new settings variable.

Reply:
This is a good point. We could defer adding `request-access` as a valid URL in the nginx config, only to be added by portals that want the route via a new entry in a supplemental `location.conf` file

Reply:
gotcha.  sounds like a good solution.

Reply:
Just confirming, no changes are needed for this review, right?

Reply:
Correct